CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Bugfixes
+- [#2448](https://github.com/influxdata/kapacitor/pull/2448): Changes the alert-handler match function duration() to be alertDuration() to avoid name collision with the type conversion function of the same name.
```

Comment:
Is this something we can change in a patch release? Or is it not really a breaking change because the collision was preventing all uses of the match function form?

Reply:
Yah, it isn't a breaking change because the old `duration()` function never actually worked.